themes/openy_themes/openy_rose/templates/node/node--branch--teaser.html.twig

```diff
@@ -80,6 +80,9 @@
     view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
   ]
 %}
+{% set directions_title = content.field_location_directions.0['#title'] %}
```

Comment:
I'm pretty sure these can result in errors if they are empty. We want to avoid the ".0" type references in twig templates. Is the directions field required when creating the node?